modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc

```diff
@@ -26,18 +28,20 @@ spec:
     - name: hello-world
       image: quay.io/baijum/secondary-workload:latest
       ports:
-      - containerPort: 9966
+      - containerPort: 8080
 ----
 
-The following example shows how to configure the `spec.containers` path by specifying a value in the `ServiceBinding` CR and to bind this path to a `spec.application.bindingPath.containersPath` custom location:
-
+[discrete]
+.Procedure
+* Configure the `spec.containers` path by specifying a value in the `ServiceBinding` CR and bind this path to a `spec.application.bindingPath.containersPath` custom location:
```

Comment:
Would it help less expert users to have the first instance of the CR in a topic spelled out, like this, "Custom Resource (CR)". If so, you could do this here and for other topics in this PR as well.